Resources/doc/form.md

```diff
@@ -13,6 +13,15 @@ $form = $formFactory->create('form', $entity);
 $form->add('file', 'fsi_file');
 ```
 
+There is only one option `'file_url'` which can be any callable with following signature:
```

Comment:
`It has only a single option - 'file_url' - which can be any callable with the following signature:`